app/migrations/Version20180102140039.php

```diff
@@ -0,0 +1,28 @@
+<?php declare(strict_types = 1);
+
+namespace Sylius\Migrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+class Version20180102140039 extends AbstractMigration
```

Comment:
missing final?

Reply:
Other migrations I checked also didn't have final. Should I add it?

Reply:
Just a suggestion :) I believe, that migration shouldn't be extended at all, but not sure about BC here ;)